app/views/questions/show.html.erb

```diff
@@ -1,16 +1,18 @@
-<p>
-  <strong>Text:</strong>
-  <%= @question.text %>
-</p>
+<% @question.translations.each do |translation| %>
+  <p>
+  <strong>Text (<%= t('language', locale: translation.locale) %>)</strong>
+  <%= translation.text %>
+  </p>
+<% end %>
 
 <p>
-  <strong>X Weight:</strong>
-  <%= @question.x_weight %>
+<strong>X Weight:</strong>
```

Comment:
Restore indentation

Reply:
Sure @spencerldixon I will restore the indentation here.